App/components/buttons/Button.tsx

```diff
@@ -1,53 +1,74 @@
 import React from 'react'
-import { StyleSheet, TouchableOpacity } from 'react-native'
+import { Text, StyleSheet, TouchableOpacity } from 'react-native'
 
-import { borderRadius, Colors } from '../../Theme'
-import Text from '../texts/Text'
+import { Colors, Buttons, ActiveOpacity } from '../../Theme'
 
-interface Props {
+export enum ButtonType {
+  Primary,
+  Secondary,
+}
+
+interface ButtonProps {
   title: string
   accessibilityLabel?: string
   onPress?: () => void
   disabled?: boolean
   neutral?: true
   negative?: true
+  buttonType?: ButtonType
 }
 
+//TODO:(jl) I think these styles should go into the button and
+//be used as an ENUM to select the look of the button.
 const styles = StyleSheet.create({
-  button: {
-    width: '90%',
-    borderRadius,
-    backgroundColor: Colors.primary,
-    alignItems: 'center',
-    padding: 10,
-    marginVertical: 10,
-  },
   disabled: {
     backgroundColor: Colors.shadow,
   },
   neutral: {
-    backgroundColor: Colors.text,
+    backgroundColor: Colors.textColor,
```

Comment:
What's the main difference between `text` and `textColor`? We probably want to stick with one across the app.

Reply:
Fixed.